src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
I think we cannot remove `navigationState` because it refers to the parent navigation state, with routes and index, rather than `navigation.state` which is the route/screen's state.

Maybe we can change the API, and one day provide wrappers to help people adapt. Nevermind for now!